main-project/src/users/entity/user.entity.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  BaseEntity,
+  Column,
+  CreateDateColumn,
+  Entity,
+  PrimaryGeneratedColumn,
+} from 'typeorm';
+
+@Entity('users')
+export class Users extends BaseEntity {
+  @PrimaryGeneratedColumn()
+  no: number;
+
+  @Column({ type: 'varchar', length: 50 })
+  email: string;
+
+  @Column({ type: 'tinyint', nullable: true })
+  gender: number;
```

Comment:
```suggestion
  @Column({ type: 'tinyint', nullable: true })
  gender: boolean;
```
유저 엔티티에서 성별은 남,녀로 쓰일 거 같은데 맞다면 불리언 타입이 맞는 거로 보여요~

Reply:
앗 감사합니다 수정하겠습니다!